src/issue.rs

```diff
@@ -10,46 +10,37 @@ use std::collections::HashMap;
 const ISSUE_NEEDS_A_PROJECT_MESSAGE: &str =
 	"{1} needs to be attached to a project or it will be closed.";
 
-fn issue_actor_and_project(
+async fn issue_actor_and_project(
 	issue: &github::Issue,
 	github_bot: &GithubBot,
 ) -> Result<Option<(github::User, github::Project)>> {
 	let repo = &issue.repository;
 	let issue_number = issue.number.context(error::MissingData)?;
-	github_bot
+	let actor_and_card = github_bot
 		.issue_events(&repo.name, issue_number)
-		.map(|issue_events| {
-			issue_events
-				.iter()
-				.sorted_by_key(|ie| ie.created_at)
-				.rev()
-				.find(|issue_event| {
-					issue_event.event == Some(github::Event::AddedToProject)
-						|| issue_event.event
-							== Some(github::Event::RemovedFromProject)
```

Comment:
We search for the most recent `AddedToProject` or `RemovedFromProject`, whichever came last, so we know if the project is currently attached or not.